client/src/components/Button/CustomButton.js

```diff
@@ -1,11 +1,12 @@
 import styled, { css } from "styled-components";
 import { Button } from "antd-mobile";
-import { ROYAL_BLUE, SELAGO } from "../../constants/colors";
-import { theme } from "../../constants/theme";
+
+import { theme, mq } from "../../constants/theme";
```

Comment:
I tried destructuring the `theme` object with `import {theme: { colors } } from "../../constants/colors"` but it caused errors in my linter. Is this possible in imports?

Reply:
seems like it's not entirely the same as object destructuring https://stackoverflow.com/questions/41137896/how-to-use-deep-destructuring-on-imports-in-es6-syntax